### PR TITLE
Add a test to retain inner packet TTL if inner packet TTL=1

### PR DESCRIPTION
**Pull request by vishnureddybadveli**
Added the missed functionality of retaining inner TTL.

### CODE REVIEW

**Comment by vishnureddybadveli** [comment]
As per vendors there is support to retain inner packet TTL, test case does not describe how exactly it is done byvendor implementation. Test case mentions that match should check for inner packet TTL == 1 but retaining inner TTL is vendor implementation.

File: feature/gribi/otg_tests/mpls_in_udp/README.md [hunk 1]
```diff
@@ -278,6 +278,133 @@ openconfig-network-instance:
...
+          encap-headers {
+            encap-header {
+              index: 2
+              type: OPENCONFIG_AFT_TYPES:UDP
+              udp {
+                src_ip: "outer_ipv6_src"
+                dst_ip: "outer_ipv6_dst_B"
+                dst_udp_port: "outer_dst_udp_port"
+                ip_ttl: "outer_ip-ttl"
+                inner_ip_ttl_min: 1
```
Comment (dplore) [inline feature/gribi/otg_tests/mpls_in_udp/README.md:389]: Can you also add your validation criteria to  [TE-18.1.2 above](https://github.com/openconfig/featureprofiles/pull/3521/files#diff-a6b35105c1accdb1889d9c529f4a13d6844d4a9844a90f3d3f736eba728502c5R200).   

Also, I think you need to define what the TTL should be on some IP traffic flow and what the action should be for each scenario.  (Define flow with TTL is 1 and another with TTL 2 and describe validation for both)

Comment (vishnureddybadveli) [reply feature/gribi/otg_tests/mpls_in_udp/README.md:389]: Thx,

Added different procedure for TTL 1 and another one for common case.

Added TTL min to TE-18.1.2.

**Comment by vishnureddybadveli** [comment]
Plan is to use policy forwarding rule instead.

**Pull request closed without merging**